documentation/business_care_intitule.md

```
@@ -0,0 +1,370 @@
+﻿*PROJET ANNUEL 2 ESGI i* 
+
+*Business Care 2024-2025* 
+
+![](Aspose.Words.29680396-ac9d-4cad-8357-49b370a89546.001.jpeg)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)